lib/middleware/websocket/client.rb

```diff
@@ -1,15 +1,20 @@
+require './lib/middleware/websocket/create_player'
+
 module Websocket
   class Client
-    attr_reader :connection, :client_attributes
+    attr_reader :connection, :uuid, :id
     attr_accessor :position
 
     def initialize(connection:)
       @connection = connection
       @position = 0
+      player = CreatePlayer.new.call
+      @uuid = player['uuid']
+      @id = player['id']
```

Comment:
I'd rather we didn't make new instance variables that can be easily derived, ie `player.uuid`. It needlessly hides where this information is coming from, which hurts readability.

Reply:
@jgayfer fixed.